engines/stark/stark.cpp

```diff
@@ -138,22 +138,16 @@ Common::Error StarkEngine::run() {
 	services.gameInterface = _gameInterface;
 	services.userInterface = _userInterface;
 	services.fontProvider = _fontProvider;
+	services.gameDescription = _gameDescription;
 
 	// Load global resources
 	_resourceProvider->initGlobal();
 	_staticProvider->init();
 	_fontProvider->initFonts();
+	
 	// Initialize the UI
 	_userInterface->init();
 
-	if (ConfMan.hasKey("save_slot")) {
-		// Load game from specified slot, if any
-		loadGameState(ConfMan.getInt("save_slot"));
```

Comment:
Ultimately, it should still be possible to load a save from the ResidualVM launcher whithout going through the main menu.